packages/gatsby/CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.24.21](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.24.20...gatsby@2.24.21) (2020-07-31)
```

Comment:
Looks like this file and its changes might be a bad merge?

Reply:
pretty sure it's this 😬 https://github.community/t/github-pull-requests-showing-invalid-diff-for-already-merged-branch/559